nqeit-iceberg-dremio-extension/src/main/java/org/projectnessie/integtests/dremio/DremioHelper.java

```diff
@@ -102,17 +115,16 @@ private String performHttpRequest(String url, String payload) throws IOException
   private void waitForJobCompletion(String jobId, String query) throws IOException {
     // The doc for getting the job status for cloud is not there, but it is similar to software
     // See docs: https://docs.dremio.com/software/rest-api/jobs/get-job/
-    String jobState = "RUNNING";
-    String url = baseUrl + "/v0/projects/" + projectId + "/job/" + jobId;
+    String url = projectUrl + "/job/" + jobId;
```

Comment:
Looks like it's worth to refactor this concatenation from three places into a helper method

Reply:
only used in 2 places, but ok